### ISSUE TITLE
Crackly audio within Ubuntu crouton

### ISSUE THREAD

**Issue by theadamgreen1** [dnschneid/crouton#791]
Within Chrome OS, sound plays fine without any issues. When it comes to playing sound through Ubuntu (installed via crouton), the sound is crackly at all volume levels. I have tried using earphones, the crackl is played through those as well. I have tried uninstalling, reinstalling, changing distribution and powerwashing my Chromebook, to no avail. I am using the Acer C7, what can I do?I never had this issue before, I uninstalled Ubuntu to try LXDE, didn't like it so reinstalled Ubuntu, to find I have crackling audio, with Unity, LXDE, GNOME and Xfce.

**Comment by stevenleeg** [comment]
I'm seeing this same issue with the latest crouton (just updated my chroot now) on debian sid (Dell Chromebook 11)

Is there anything I can provide to help fix this?

**Comment by Judecca** [comment]
I have exactly the same issue. Audio crackles on Ubuntu XFCE. Most notably, in VLC or Audacity. I have the most recent version of Crouton and just updated my chroot. 

Acer Chromebook 11 CB3-111

**Comment by concreteface** [comment]
Same thing for me. Running Trusty (KDE) on a c720. Any headway on this?

**Comment by yetzke** [comment]
Same thing for me. I'm running Precise KDE on c720. Thanks

**Comment by forresthopkinsa** [comment]
Bump. I'm having the same problem. It's not consistent, though--sometimes it happens, sometimes not. But when it starts happening, it gets progressively worse unless I stop the source of sound for a few minutes. That sometimes fixes it, sometimes not. I just updated, and that didn't fix it. This has been happening for months. It seems to happen a lot more often through an external audio output, like HDMI or earphones. It's more rare through the internal speakers.

HP Chromebook 14, 2013
Ubuntu 14.04

**Comment by ee7klt** [comment]
can someone please explain how exactly to do this in layman's terms? I've clicked through the link to #1053 by drinkcat and the subsequent links from there. From what I've gleaned, it looks like we need to set the version number in CPPflags or something or other. What exact commands do i need to run? Will I need to re-install ubuntu? if i need to copy libcras.so from chrome os to ubuntu, what is the origin and target of this file?

**Comment by forresthopkinsa** [comment]
Honestly, it eventually went away for me after an update. Then again, I don't use the external HDMI output nearly as often as I used to, so maybe it has only dropped in frequency.

**Comment by AdamDanischewski** [comment]
This problem is still existent, everything was fine until recently (over the last week -- I noticed an audio problem a few days ago yet didn't reboot because of ongoing programs).  

$> aplay -Dcras /usr/share/sounds/alsa/Front_Center.wav 
works clear and fine. 

Playing audio through vlc w/Chromium OS Audio Server or Default produces horrible scratchy and screetchy sounds.

$> cras_test_client --version
crouton-fe461bde8bb9095106ea8ac67b0701a366d8180

crouton: version 1-20160902144033~master:9f8e9a22
release: trusty
architecture: amd64
xmethod: xorg
targets: xfce
host: version 8530.81.0 (Official Build) stable-channel peppy 
kernel: Linux localhost 3.8.11 #1 SMP Wed Sep 7 17:09:28 PDT 2016 x86_64 x86_64 x86_64 GNU/Linux
freon: yes

VLC media player 2.1.6 Rincewind (revision 2.1.6-0-gea01d28)
VLC version 2.1.6 Rincewind (2.1.6-0-gea01d28)
Compiled by buildd on lgw01-55.buildd (May  3 2016 01:05:31)
Compiler: gcc version 4.8.2 (Ubuntu 4.8.2-19ubuntu1)
This program comes with NO WARRANTY, to the extent permitted by law.
You may redistribute it under the terms of the GNU General Public License;
see the file named COPYING for details.
Written by the VideoLAN team; see the AUTHORS file.

pulseaudio 4.0

**Comment by forresthopkinsa** [comment]
you're on the latest commit, so that isn't the issue. weird.

**Comment by forresthopkinsa** [comment]
@dnschneid Do you have to update Crouton to work with Cras with each Chromium update?
